Account Onboard Utility/README.md

```diff
@@ -61,6 +61,15 @@ Accounts_Onboard_Utility.ps1 -PVWAURL <string> [-<Create / Update / Delete>] [-A
 - CreateOnUpdate
 	- By default, the script will automatically not create new accounts when in update mode
 	- Using this switch will automatic create accounts that do not exist when running in update mode
+- ConcurrentSession
+	- By default, Any sessions logged into will be disconnected.
+	- Using this switch will allow for Concurrent Sessions for the user. This includes additional REST API calls (Which must also be set to ConcurrentSession) or allow for connected PVWA user sessions to remain.
+- BypassSafeSearch
+	- By default, the script will automatically search to see if the account exists or if it needs to be created
+	- Using this switch in create/update mode well prevent safe searches, but may result in account operations failures if the safe does not exist. This should only be used when it is known all safes listed already exist. USE WITH EXTREME CAUTION. 
+- BypassAccountSearch
+	- By default, the script will automatically search for requested accounts to determine if they exist. This search is done via "name" property or "username" and "address" if name is not present
+	- Using this switch in create mode will assume that the account does not exist and will attempt to create them. If the "name" property is not populated, there is no checking for duplicate accounts. If the name property is populated only duplicate "name" properties will be detected and will cause a failure. All other scenarios MAY result in duplicates including if only "username" and "address" properties are populated as this will cause "name" field to be auto populated and automatically incremented up by 1. USE WITH EXTREME CAUTION. 
```

Comment:
There could be duplications also if name is not populated (automatically)
Maybe we should rephrase it in general that duplications might be created if using this switch so use with extreme caution

Reply:
I updated the working.